tests/parm/v16_c96_rrtmgp.nml.IN

```diff
@@ -41,14 +41,14 @@ deflate_level=1
 /
 
 &fv_core_nml
-  layout = @[INPES],@[JNPES]
+  layout = 3,8
```

Comment:
Why this change, isn't `3,8` the default value in `default_vars.sh` ?

Reply:
reverted.